presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -657,7 +658,12 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         USE_RECORD_PAGE_SOURCE_FOR_CUSTOM_SPLIT,
                         "Use record page source for custom split",
                         hiveClientConfig.isUseRecordPageSourceForCustomSplit(),
-                        false));
+                        false),
+                integerProperty(
+                        MAX_INITIAL_SPLITS,
+                        "Hive Max Initial Split Size",
```

Comment:
lower cases + typo in the end: "Hive max initial split count"

Reply:
thanks, have fixed it